examples/food-imports/src/components/Sunburst.js

```diff
@@ -28,7 +28,7 @@ export function Sunburst(
     startAngle = 0, // the starting angle for the sunburst
     endAngle = 2 * Math.PI, // the ending angle for the sunburst
     radius = Math.min(width - marginLeft - marginRight, height - marginTop - marginBottom) / 2, // outer radius
-    color = d3.interpolateRainbow, // color scheme, if any
+    color = d3.interpolateRainbow, // color scheme or scale, if any
```

Comment:
I'm not sure this is the right approach—we need to make sure the same color scale is used here and in the marimekko